start-mycroft.sh

```diff
@@ -144,13 +144,21 @@ function launch-all() {
 }
 
 function check-dependencies() {
-  if [ ! -f .installed ] || ! md5sum -c &> /dev/null < .installed ; then
-    echo "Please update dependencies by running ./dev_setup.sh again."
-    if command -v notify-send >/dev/null ; then
-      notify-send "Mycroft Dependencies Outdated" "Run ./dev_setup.sh again"
+    if [ ! -f .installed ] || ! md5sum -c &> /dev/null < .installed ; then
+        # Critical files have changed, dev_setup.sh should be run again
+        auto_update=$( jq -r ".auto_update" < .dev_opts.json )
```

Comment:
Maybe add 2> /dev/null for the time being since this shows file not found error messages right now.

Reply:
Fixed